resolwe_bio/processes/import_data/tabular.yml

```diff
@@ -0,0 +1,50 @@
+# ===================
+# Import tabular file
+# ===================
+---
+
+- slug: upload-tab-file
+  name: Tabular file
+  data_name: '{{ src.file|default:"?" }}'
+  version: 0.0.1
+  type: data:file:tab
+  category: upload
+  persistence: RAW
+  description: >
+    Upload tabular file.
+  input:
+    - name: src
+      label: CSV file
+      type: basic:file
+      description: >
+        Tabular file. Supported extensions: .csv, .csv.gz (preferred) or .csv.*
+      validate_regex: '(\.(csv|tab|tsv|txt|xls|xlsx)(|\.gz|\.bz2|\.tgz|\.tar\.gz|\.tar\.bz2|\.zip|\.rar|\.7z))|(\.bz2)$'
+  output:
+    - name: file
+      label: Tabular file
+      type: basic:file
+  run:
+    runtime: polyglot
+    bash: |
+      if [[ '.{{ src.file }}' =~ \.(xls)$ ]]; then
```

Comment:
Re-import not needed unless we wish the file to be zipped. Do we? Just move the original file and then call `parse_tabular_file.py`. Save both, the original file and the parsed file.